zapcore/level.go

```diff
@@ -183,7 +183,7 @@ func (l *Level) unmarshalText(text []byte) bool {
 		*l = DebugLevel
 	case "info", "INFO", "": // make the zero value useful
 		*l = InfoLevel
-	case "warn", "warning", "WARN":
+	case "warn", "warning", "WARN", "WARNING":
```

Comment:
is this actually needed? unmarshalText is called with `bytes.ToLower` at some point.

Reply:
yea i just saw that we call it two times, once with the text itself and another time with ToLower()... 

In this case I think we should just remove all the capitalized cases in unmarshalText since they're redundant.